Cython/Utility/CMath.c

```diff
@@ -93,3 +93,20 @@ static CYTHON_INLINE %(type)s %(func_name)s(%(type)s b, %(type)s e) {
     }
     return t;
 }
+
+
+/////////////// IsUnsigned.proto ///////////////
```

Comment:
I think `ModuleSetupCode.c` is a better place for this (`CMath.c` definitely is not), maybe before the `CInitCode` section.